templates/snapcraft.yaml.j2

```diff
@@ -61,12 +62,21 @@ package-repositories:
     type: apt
 {%- if versionmap[ros_distro] == '1' %}
     url: http://packages.ros.org/ros/ubuntu
+{%- elif ros_distro == 'jazzy' %}
+    url: http://packages.ros.org/ros2-testing/ubuntu
```

Comment:
This can be removed now.

Reply:
Done in [66f12b5](https://github.com/ubuntu-robotics/ros-content-sharing-snaps/pull/9/commits/66f12b5e6a928ee3b1d3f5bdd69647caa16da6f1)